Skyra.Docker/docker-compose.yml

```diff
@@ -2,23 +2,24 @@ version: "3.7"
 services:
   rabbitmq:
     container_name: rabbitmq
-    image: rabbitmq:management-alpine
+    image: rabbitmq:alpine
     restart: always
     ports:
-      - '15672:15672'
-      - '5672:5672'
+      - "5672:5672"
   redis:
     container_name: redis
-    image: skyrabot/redis:latest
+    image: redis:alpine
```

Comment:
No. Then we cannot set a custom password. See the Redis directory. And before you say "it failed to start", that's because you didn't build locally it and it wasn't pushed to dockerhub yet. Either build it with the docker-build.yml in rider or run the PS1 with build -> Redis.

Reply:
Then push to dockerhub please, once you do so I'll revert this one